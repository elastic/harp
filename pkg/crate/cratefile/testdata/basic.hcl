# Required
container "region-eu" {
    # Path to the container.
    path = "region-eu.bundle"

    # Idetities for sealing purpose if the container is not sealed.
    identities = [
        "v1.ipk.7u8B1VFrHyMeWyt8Jzj1Nj2BgVB7z-umD8R-OOnJahE", # Security public key
    ]
}

# Optional archive layer
# All files matching the filter will be compressed as a tar.gz and embedded
# in the crate.
archive "production" {
    # Root path from where files are crawled to create the archive.
    root = "./production"

    # Include filters.
    includes = [
        "**"
    ]

    # Exclude filters.
    excludes = [
        "**.go"
    ]
}
