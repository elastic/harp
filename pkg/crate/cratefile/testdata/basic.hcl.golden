(*cratefile.Config)({
 Container: (cratefile.Container) {
  Name: (string) (len=9) "region-eu",
  Path: (string) (len=16) "region-eu.bundle",
  Identities: ([]string) (len=1 cap=1) {
   (string) (len=50) "v1.ipk.7u8B1VFrHyMeWyt8Jzj1Nj2BgVB7z-umD8R-OOnJahE"
  }
 },
 Archives: ([]cratefile.Archive) (len=1 cap=1) {
  (cratefile.Archive) {
   Name: (string) (len=10) "production",
   RootPath: (string) (len=12) "./production",
   IncludeGlob: ([]string) (len=1 cap=1) {
    (string) (len=2) "**"
   },
   ExcludeGlob: ([]string) (len=1 cap=1) {
    (string) (len=5) "**.go"
   }
  }
 }
})
